<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyWindow</class>
 <widget class="QMainWindow" name="EasyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Typing Speed Test- Easy</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icon.png</normaloff>../assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="easy_input_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>440</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="easyLineEdit">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>90</y>
       <width>430</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLowercaseOnly|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="easy_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>420</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Placeholder text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="easy_title_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>400</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Type out the following word down below:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="easy_button_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="easy_restart">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Restart</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
